### ISSUE TITLE
Toast-Swift not working in Xcode15, swift 5.9

### ISSUE THREAD

**Issue by cuongpham2110** [scalessec/Toast-Swift#204]
Hi, 
I use this library in the past and it worked perfectly in Xcode 12.
Now when I upgrade to Xcode15, toast is show up but is not dismissed automatically even in case of calling hideAllToasts()
Is there anyone meet such bug? 
Thanks for any help

**Comment by uyloal** [comment]
To resolve the issue, you need to modify the data type of `ToastKeys` and change it to `Int`.
```swift
    private struct ToastKeys {
        static var timer        = "com.toast-swift.timer".hash
        static var duration     = "com.toast-swift.duration".hash
        static var point        = "com.toast-swift.point".hash
        static var completion   = "com.toast-swift.completion".hash
        static var activeToasts = "com.toast-swift.activeToasts".hash
        static var activityView = "com.toast-swift.activityView".hash
        static var queue        = "com.toast-swift.queue".hash
    }
```

**Comment by scalessec** [comment]
Fixed in `5.1.0`.
